examples/CMakeLists.txt

```diff
@@ -85,23 +85,39 @@ add_subdirectory(api/cpp)
 
 if(TARGET cvc5::cvc5jar)
   find_package(Java REQUIRED)
+  find_package(JNI REQUIRED)
   include(UseJava)
+  message(STATUS "JNI_LIBRARIES: ${JNI_LIBRARIES}")
 
-  ## disabled until bindings for the new API are in place (issue #2284)
-  # get_target_property(CVC5_JAR cvc5::cvc5jar JAR_FILE)
-  #
-  # add_jar(SimpleVC SimpleVC.java INCLUDE_JARS "${CVC5_JAR}")
-  #
-  # add_test(
-  #   NAME java/SimpleVC
-  #   COMMAND
-  #     ${Java_JAVA_EXECUTABLE}
-  #       -cp "${CVC5_JAR}:${CMAKE_BINARY_DIR}/SimpleVC.jar"
-  #       -Djava.library.path=${CVC5_JNI_PATH}
-  #       SimpleVC
-  # )
-  # TODO(project wishue #83): enable java examples
-  # add_subdirectory(api/java)
+  # get the directories of libraries libjawt.so and libjvm.so
+  set(JNI_LIBRARIES_PATHS "")
+  foreach(LIB ${JNI_LIBRARIES})
+    get_filename_component(LIB_PATH ${LIB} DIRECTORY)
+    set(JNI_LIBRARIES_PATHS "${JNI_LIBRARIES_PATHS}${LIB_PATH}:")
+  endforeach()
+
+  message(STATUS "JNI_LIBRARIES_PATHS: ${JNI_LIBRARIES_PATHS}")
+  # get directory build/install/lib where libcvc5jni.so is installed
+  get_target_property(CVC5_LIB_FILE cvc5::cvc5 LOCATION)
+  get_filename_component(CVC5_JNI_PATH ${CVC5_LIB_FILE} DIRECTORY)
```

Comment:
This works for now, because libcvcjni is installed in the same directory of libcvc5.so. 
It would fail if there are installed in different directories. 
Is there a simple way to get a path to `build/install/lib` for libcvcjni5 without adding exporting it in a cmake file?